CARTOGRAPHY.md

```diff
@@ -37,3 +37,67 @@ account.
 * **A rich map** - This style deliberately creating a fairly rich map showing a significant number of different features.  This way it shows the richness of OSM data and gives a broad recognition to the mappers' work.  The aim is not however to show all or even most of the OSM data.
 * **Maintainability** - The implementation of this style should not be too hard to maintain. This refers to both the volume and complexity of the code and how fast the style can be parsed when rendering it, which is very important for efficient development work. So the amount of code should be kept small and complex and fragile interdependencies should be avoided. If the code is difficult to maintain this would ultimately seriously affect all of the above goals.
 * **Adaptability and ease of use** - The style should be easy to customize, like for creating localized or special purpose maps. It is also important to keep demands on rendering infrastructure for serving the style low so it is not too difficult and costly to set up a tile server for this style or a specialized variant of it.
+
+## Design guidelines
+
+To accomplish these goals the following guidelines have been found to be useful.
+
+Decisions on if and how to display something in the map should be primarily based on the significance of features for the purposes and goals above.
+
+* How often something is currently mapped in OSM has relatively little influence on such decisions.  Widespread use of a tag can be a supporting factor but should never be the reason for rendering something.  Tag use also needs to be seen in relation to the actual occurrence of the type of feature in reality.  And rare use of a tag of course means there is very limited information to assess the quality of the data.
+* If mapping in OSM (i.e. use of tags and geometric representation) is consistent, verifiable and accurate is highly relevant.  Rendering something that is mapped inconsistently is counterproductive for all of this style's purposes.  Consistency is to be looked for both in a positive way (mapping with the tag in question is consistent) and in a negative way (the same kind of object in reality is not frequently mapped in a different way).  Different competing tags can be accepted (to be rendered in the same way) if both variants are well established and the difference mainly exists for historic reasons and their use is compatible.
```

Comment:
I get **some** of what this is trying to say, having worked with landuse=forest and natural=wood. I don't think most people will understand it.

I'm not sure if I agree with it as phrased.